src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -51,17 +51,40 @@ class RsHighlightingAnnotator : AnnotatorBase() {
         }
 
         return when (element.elementType) {
+            DOLLAR -> RsColor.MACRO // TODO: It would be possible to see whether parent is RsMacroBindingGroups or RsMacroExpansionReferenceGroup or RsMacroBinding
             IDENTIFIER, QUOTE_IDENTIFIER, SELF -> highlightIdentifier(element, parent, holder)
             // Although we remap tokens from identifier to keyword, this happens in the
             // parser's pass, so we can't use HighlightingLexer to color these
             in RS_CONTEXTUAL_KEYWORDS -> RsColor.KEYWORD
             FLOAT_LITERAL -> RsColor.NUMBER
 
-            Q -> if (parent is RsTryExpr) {
+            Q -> if (parent is RsMacroExpansionReferenceGroup || parent is RsMacroBindingGroup) { // TODO: and it the last non-whitespace element in parent
```

Comment:
I did the first, but I don't understand what you mean by the second.

I pushed it, you can take a look.

Reply:
I meant that you can do the following:
```kotlin
MUL, PLUS, LPAREN, LBRACE, RPAREN, RBRACE -> macroGroupColor(parent, element)
```
instead of the same separate branches

Reply:
Done.